lib/iris/fileformats/grib/_grib_cf_map.py

```diff
@@ -190,33 +193,37 @@
     CFName('lagrangian_tendency_of_air_pressure', None, 'Pa s-1'): G2Param(2, 0, 2, 8),
     CFName('land_area_fraction', None, '1'): G2Param(2, 2, 0, 0),
     CFName('land_binary_mask', None, '1'): G2Param(2, 2, 0, 0),
+    CFName('land_binary_mask', None, '1'): G2Param(2, 2, 0, 0),
     CFName('liquid_water_content_of_surface_snow', None, 'kg m-2'): G2Param(2, 0, 1, 13),
     CFName('low_type_cloud_area_fraction', None, '%'): G2Param(2, 0, 6, 3),
     CFName('medium_type_cloud_area_fraction', None, '%'): G2Param(2, 0, 6, 4),
     CFName('moisture_content_of_soil_layer', None, 'kg m-2'): G2Param(2, 2, 0, 22),
     CFName('precipitation_amount', None, 'kg m-2'): G2Param(2, 0, 1, 49),
+    CFName('precipitation_amount', None, 'kg m-2'): G2Param(2, 0, 1, 52),
     CFName('precipitation_flux', None, 'kg m-2 s-1'): G2Param(2, 0, 1, 7),
     CFName('relative_humidity', None, '%'): G2Param(2, 0, 1, 1),
     CFName('sea_ice_area_fraction', None, '1'): G2Param(2, 10, 2, 0),
     CFName('sea_surface_temperature', None, 'K'): G2Param(2, 10, 3, 0),
     CFName('sea_water_x_velocity', None, 'm s-1'): G2Param(2, 10, 1, 2),
     CFName('sea_water_y_velocity', None, 'm s-1'): G2Param(2, 10, 1, 3),
+    CFName('snowfall_amount', None, 'kg m-2'): G2Param(2, 0, 1, 60),
     CFName('snowfall_flux', None, 'kg m-2 s-1'): G2Param(2, 0, 1, 53),
     CFName('soil_temperature', None, 'K'): G2Param(2, 2, 0, 2),
     CFName('specific_humidity', None, 'kg kg-1'): G2Param(2, 0, 1, 0),
     CFName('surface_air_pressure', None, 'Pa'): G2Param(2, 0, 3, 0),
+    CFName('surface_air_pressure', None, 'Pa'): G2Param(2, 0, 3, 0),
     CFName('surface_altitude', None, 'm'): G2Param(2, 2, 0, 7),
     CFName('surface_downwelling_longwave_flux_in_air', None, 'W m-2'): G2Param(2, 0, 5, 3),
     CFName('surface_downwelling_shortwave_flux_in_air', None, 'W m-2'): G2Param(2, 0, 4, 7),
     CFName('surface_net_downward_longwave_flux', None, 'W m-2'): G2Param(2, 0, 5, 5),
+    CFName('surface_net_downward_shortwave_flux', None, 'W m-2'): G2Param(2, 0, 5, 5),
```

Comment:
This CFName (`surface_net_downward_shortwave_flux`) appears to be associated with two G2Params (on this line and the next line). The G2Param it is associated with is **also** duplicated from the line above.